go.mod

```diff
@@ -1,21 +1,18 @@
-module github.com/anz-bank/sysl-go-comms
+module github.com/anz-bank/sysl-go
 
 require (
+	github.com/anz-bank/sysl-go-comms v0.0.0-20200313054710-9d2033a94714 // indirect
```

Comment:
This import should be removed.

Reply:
Yup thanks, removed in f62e203d25a8344b3efc3feffa11dc740d095ef1